distribution/docker/src/docker/bin/docker-entrypoint.sh

```diff
@@ -57,6 +57,21 @@ if [[ -f bin/elasticsearch-users ]]; then
   fi
 fi
 
+if [[ -n "$ES_LOG_STYLE" ]]; then
+  case "$ES_LOG_STYLE" in
+    docker)
+      # This is the default. Nothing to do.
+      ;;
+    stack)
```

Comment:
I'm not in love with this name. It doesn't really mean anything from a user perspective. I'd be more inclined to call this `file` or `disk`.

Reply:
Agreed, I've swapped to `console` and `file.